main.go

```diff
@@ -40,6 +40,7 @@ import (
 	metricsserver "sigs.k8s.io/controller-runtime/pkg/metrics/server"
 
 	networkv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+	operatorv1 "github.com/openshift/api/operator/v1"
```

Comment:
do we need this right now?

Reply:
but line 101 would not watch it. it just adds the permissions to be able to watch it. In my opinion we should not watch objects we don't use right now. we should add it later.

Reply:
Oh true, I forgot to add the watch part when I was copying the code. Instead of watching I will add a //TODO that if clusterLocal is parameterizable we need to watch the resource